frontend/src/pages/modelServing/utils.ts

```diff
@@ -114,3 +131,51 @@ export const getTokenNames = (servingRuntimeName: string, namespace: string) =>
 
   return { serviceAccountName, roleBindingName };
 };
+
+export const getServingRuntimeSize = (
+  sizes: ServingRuntimeSize[],
+  servingRuntime?: ServingRuntimeKind,
+): ServingRuntimeSize => {
+  const existingResources = servingRuntime?.spec?.containers[0]?.resources || sizes[0].resources;
+  const size = sizes.find(
+    (size) =>
+      isCpuLimitEqual(size.resources.limits?.cpu, existingResources.limits?.cpu) &&
+      isMemoryLimitEqual(size.resources.limits?.memory, existingResources.limits?.memory),
+  );
+  return (
+    size || {
+      name: 'Custom',
+      resources: existingResources,
+    }
+  );
+};
+
+export const getServingRuntimeTokens = (tokens?: SecretKind[]): ServingRuntimeToken[] =>
+  (tokens || []).map((secret) => ({
+    name: getDisplayNameFromK8sResource(secret) || secret.metadata.name,
+    editName: secret.metadata.name,
+    uuid: secret.metadata.name,
+    error: '',
+  }));
+
+export const isModelServerEditInfoChanged = (
```

Comment:
this needs to check for accelerator dropdown now too.

Reply:
Updated, could you check it again?